src/clientManager.js

```diff
@@ -52,7 +51,6 @@ ClientManager.prototype.loadUser = function(name) {
 
 ClientManager.prototype.getUsers = function() {
 	var users = [];
-	mkdirp.sync(Helper.USERS_PATH);
```

Comment:
Since you removed these syncs from clientManager, can you add one in `command-line/index.js` after ensuring `CONFIG_PATH` folder?